File: Map/src/app/shared/DanhMucCuaHang.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucCuaHang } from 'src/app/shared/DanhMucCuaHang.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucCuaHangService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucCuaHang";
    }
}

File: Service/Implement/DanhMucCuaHangService.cs
namespace Service.Implement
{
    public class DanhMucCuaHangService : BaseService<DanhMucCuaHang, IDanhMucCuaHangRepository>
    , IDanhMucCuaHangService
    {
    private readonly IDanhMucCuaHangRepository _DanhMucCuaHangRepository;
    public DanhMucCuaHangService(IDanhMucCuaHangRepository DanhMucCuaHangRepository) : base(DanhMucCuaHangRepository)
    {
    _DanhMucCuaHangRepository = DanhMucCuaHangRepository;
    }
    }
    }

